src/System.Net.Http/src/System/Net/Http/NetEventSource.Http.cs

```diff
@@ -65,6 +66,19 @@ public static void ClientSendCompleted(HttpClient httpClient, HttpResponseMessag
             WriteEvent(HandlerMessageId, handlerId, workerId, requestId, memberName, message);
             //Console.WriteLine($"{handlerId}/{workerId}/{requestId}: ({memberName}): {message}");  // uncomment for debugging only
 
+        [NonEvent]
+        public static void AuthenticationInfo(Uri uri, string message)
+        {
+            if (IsEnabled)
```

Comment:
The format established in this file is to always wrap the call to `Log` in an `IsEnabled` check. However I am still checking that logging is enabled before each call to `AuthenticationInfo`. That means we do the check twice in the case logging is enabled, but avoid the cost of allocating and formatting strings when logging is off.

I think we should probably just update the format to keep all checks to `IsEnabled` outside of these calls, to avoid duplicate work.

Reply:
>I think we should probably just update the format to keep all checks to IsEnabled outside of these calls, to avoid duplicate work.

If you're always checking IsEnabled before calling the AuthenticationInfo() methods, then change the 'if (IsEnabled)' to a 'Debug.Assert(IsEnabled)'.

Reply:
That sounds good to me.